common/crypto/dkg/src/bte/keys.rs

```diff
@@ -414,395 +292,31 @@ impl PublicKeyWithProof {
 #[zeroize(drop)]
 #[cfg_attr(test, derive(PartialEq))]
 pub struct DecryptionKey {
-    // note that the nodes are ordered from "right" to "left"
-    pub(crate) nodes: Vec<Node>,
+    pub(crate) inner: Node,
```

Comment:
I think we could further simplify it and remove `Node` struct (by moving everything from there into the `DecryptionKey`; I **think**, but I'm not 100% sure, you might be able to remove `Tau` from `Node` because it's always going to be constant in our case)